### PR TITLE
Add discord-bots topic

### PR DESCRIPTION
**Pull request by angrymouse**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> I added discord-bots topic

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [x] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Add discord-bots topic

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Review by kenyonj — CHANGES_REQUESTED** [review]
Looks like the image file has to be renamed to match the topic name. Once that is complete CI will likely be green and we can get this merged.

**Comment by angrymouse** [comment]
Done!

File: topics/discord-bots/index.md
Comment (J-Human) [inline topics/discord-bots/index.md:7]:
```suggestion
short_description: Discord Bots are extensions for the Discord chat app.
```
Just a small grammar correction.

File: topics/discord-bots/index.md
Comment (J-Human) [inline topics/discord-bots/index.md:10]:
```suggestion
A bot is an automated Discord account. It utilizes the Discord API. They have a "BOT" tag next to their username. They can be added through the API. Bots typically follow a command structure, where a user sends a prefixed message or a slash command, and the bot reponds, though bots can work in many different ways.
```
I think it would be more consistent to use the American spelling of things (like the words in other pages, e.g. "color"). Second, slash commands are also dominant in today's era of Discord [bots]. Third, of course, the Oxford comma before the "and" 😄.

Comment (angrymouse) [reply topics/discord-bots/index.md:10]: Thank you for correcting me! <3

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

File: topics/discord-bots/index.md
Comment (kenyonj) [inline topics/discord-bots/index.md:8]: I don't feel comfortable linking to a 3rd party bot listing site here. Can we remove this line?
```suggestion
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
Apologies, just noticed one more thing that I think we need to change before merging it in.

File: topics/discord-bots/index.md
Comment (angrymouse) [reply topics/discord-bots/index.md:8]: Let's wait for the discord to release its own listing service (it will do it in about a month or two)

Comment (angrymouse) [reply topics/discord-bots/index.md:8]: Or we can for now replace it with https://discord.dev, which is an official documentation of discord's api

Comment (kenyonj) [reply topics/discord-bots/index.md:8]: sure, yeah that works!
```suggestion
url: https://discord.dev
```

**Review by kenyonj — APPROVED** [review]

**Pull request merged**
